package.json

```diff
@@ -92,7 +92,9 @@
     "prettify": "prettier --write '**/*.js'",
     "check-pretty": "prettier --list-different '**/*.js'",
     "start": "NODE_ENV=development webpack-dev-server --config config/webpack.config.web.js",
+    "start2": "NODE_ENV=development webpack-dev-server --config config/webpack.config.web2.js",
```

Comment:
How long will we care about backwards compatibility? Do we want to be maintaining two versions?

Reply:
We'll throw away the old version as soon as the new one can take over. The old stuff is all crap IMO.